src/libs/web3/web3.connectors.ts

```diff
@@ -136,3 +152,84 @@ export const coinbaseWalletConnection: Connection = {
   name: 'Coinbase Wallet',
   logoUrl: iconCoinbase,
 };
+
+// ********************************** //
+// Tailwind WALLET CONNECTOR
+// ********************************** //
+
+const [web3TailwindWallet, web3TailwindWalletHooks] =
+  initializeConnector<TailwindConnector>(
+    (actions) => new TailwindConnector({ actions })
+  );
+
+export const tailwindWalletConnection: Connection = {
+  connector: web3TailwindWallet,
+  hooks: web3TailwindWalletHooks,
+  type: 'tailwindWallet',
+  name: 'Tailwind Wallet',
+  logoUrl: iconTailwindWallet,
+};
+
+// ********************************** //
+// Compass WALLET CONNECTOR
+// ********************************** //
+
+export const [web3CompassWallet, web3CompassWalletHooks] = getInjectedProvider(
```

Comment:
I don't get it - are we initializing the connectors in any case?

Reply:
As Doron says, I would recommend exporting a factory function and use it only when you need it. This would reduce CPU usage a launch time.